apps/monitoring/dynatrace/kustomization.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - dynatrace-cluster-role.yaml
+  # - dynatrace-cluster-role.yaml
```

Comment:
It actually comes from here:
https://github.com/hmcts/aks-sds-deploy/blob/master/bootstrap/dynatrace-cluster-role-binding.yaml

Reply:
better to remove from bootstrap instead?